File: rb/utility/make_context.rb
# OgliLinkShortener SDK utility: make_context
require_relative '../core/context'
module OgliLinkShortenerUtilities
  MakeContext = ->(ctxmap, basectx) {
    OgliLinkShortenerContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// OgliLinkShortener SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class OgliLinkShortenerMakeContext
{
    public static function call(array $ctxmap, ?OgliLinkShortenerContext $basectx): OgliLinkShortenerContext
    {
        return new OgliLinkShortenerContext($ctxmap, $basectx);
    }
}
